src/utils/mtev_watchdog.c

```diff
@@ -548,6 +555,7 @@ int mtev_watchdog_start_child(const char *app, int (*func)(void),
       sigaddset(&mysigs, SIGINT);
       setup_signals(&mysigs);
       mtev_monitored_child_pid = child_pid;
+      mtev_conf_get_boolean(MTEV_CONF_ROOT, "//watchdog/@save_trace_output", &save_trace_output);
```

Comment:
we should not use mtev_conf from with utils/

We need to expose a function to set that boolean and call it from mtev_main.c dependent upon this config value

Reply:
Addressed in latest commit.